### PR TITLE
feat(spotify-web): init

### PR DESCRIPTION
**Pull request by Tnixc**
## 🎉 Theme for Spotify Web 🎉

A theme for the web version of spotify!

## 💬 Additional Comments 💬

There are some svgs that I couldn't figure out how to theme.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

File: styles/spotify-web/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+/* If you need any help with the information above, please see: https://github.com/openstyles/stylus/wiki/Writing-UserCSS#var */
+
+/* Apply Theme To Site
+ * For help with URL rules, please see https://github.com/openstyles/stylus/wiki/Writing-styles#valid--moz-document-rules */
+@-moz-document regexp('https://open\.spotify\.com/.*')
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:20]: You could probably just do a `@-moz-document domain('open.spotify.com*')`.

File: styles/spotify-web/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+   *  populated through the `@var`s in the Meta Block. */
+  // synced colorscheme
+
+  @media (prefers-color-scheme: light) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  @media (prefers-color-scheme: dark) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+} /* /@-moz-document */
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:395]: You can remove this `/* /@-moz-document */` comment at the end, it was a mistake in the template I think.

File: styles/spotify-web/catppuccin.user.css [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 /* Apply Theme To Site
  * For help with URL rules, please see https://github.com/openstyles/stylus/wiki/Writing-styles#valid--moz-document-rules */
-@-moz-document regexp('https://open\.spotify\.com/.*')
+@-moz-document domain('open.spotify.com*')
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:20]: Sorry make sure to remove the star at the end there.

File: styles/spotify-web/catppuccin.user.css [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+    .RVRoa p {
+      color: @text;
+    }
+    // Social buttons
+    .kmZeYl:hover {
+      background-color: @crust;
+    }
+    .kmZeYl {
+      background-color: @mantle;
+    }
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:342]:
```suggestion
    .kmZeYl {
      background-color: @mantle;
      &:hover {
        background-color: @crust;
      }
    }
```

File: styles/spotify-web/catppuccin.user.css [hunk 5]
```diff
@@ -0,0 +1,395 @@
+/* ==UserStyle==
+@name spotify-web Catppuccin
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:2]:
```suggestion
@name Spotify Web Catppuccin
```

File: styles/spotify-web/catppuccin.user.css [hunk 6]
```diff
@@ -0,0 +1,395 @@
...
+@namespace github.com/catppuccin/userstyles/styles/spotify-web
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/spotify-web
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/spotify-web/catppuccin.user.css
+@description Soothing pastel theme for spotify-web
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:7]:
```suggestion
@description Soothing pastel theme for Spotify Web
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
Sorry I probably should have done that in one suggestion :D

**Review by nekowinston — CHANGES_REQUESTED** [review]
The screenshots are currently not stored with LFS, so I couldn't resolve the latest merge from main for you.

File: styles/spotify-web/catppuccin.user.css [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+      --background-press: @surface1;
+      --background-elevated-base: @surface0;
+      --background-elevated-highlight: @surface1;
+      --background-elevated-press: @crust;
+      --background-unsafe-for-small-text-base: @base;
+      --background-unsafe-for-small-text-highlight: @base;
+      --background-unsafe-for-small-text-press: @base;
+      --background-unsafe-for-small-text-base: @base;
+      --background-unsafe-for-small-text-highlight: @base;
+      --background-unsafe-for-small-text-press: @base;
```
Comment (isabelroses) [inline styles/spotify-web/catppuccin.user.css:110]: Can you look over these duplicated properties

**Review by isabelroses — CHANGES_REQUESTED** [review]
<details>
<summary>
Hover items in general seem to be broken.
</summary>

image
image
image
image
image

</details>
<details>
<summary>
This popup seems too be the wrong colour
</summary>

image

</details>
<details>
<summary>
The search library button seems to be broken
</summary>

image

</details>

**Review by nekowinston — APPROVED** [review]

File: styles/spotify-web/catppuccin.user.css [hunk 8]
```diff
@@ -0,0 +1,368 @@
...
+    }
+  }
+
+  @media (prefers-color-scheme: light) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  @media (prefers-color-scheme: dark) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+}
```
Comment (isabelroses) [inline styles/spotify-web/catppuccin.user.css:368]:
```suggestion
}
// vim:ft=less
```

**Review by isabelroses — APPROVED** [review]

**Pull request merged**